src/App.js

```diff
@@ -1,14 +1,36 @@
-import React, { Component } from 'react';
-import PostCommentsDisplay from './PostCommentsDisplay/components/PostCommentsDisplay';
+import React from 'react';
+import { Router, Link } from '@reach/router';
+
+import Home from './Home/components/Home';
+import MakeListingDisplay from './MakeListingDisplay/components/MakeListingDisplay';
+
 import './App.css';
 
-/**Component class to render the whole comments page */
-class App extends Component {
-    render(){
-        return (
-            <PostCommentsDisplay />
-        );
-    }
+/** Component class to render the whole comments page*/
```

Comment:
Sorry should have been clear before. Also add space before end of the comment. (Here and other places)
'page */'

Reply:
Done